camel/societies/workforce/workforce_logger.py

```
@@ -0,0 +1,613 @@
+
+        # Add the current task
+        result[task_id] = self._task_hierarchy[task_id]
+
+        # Add all children recursively
+        children = self._task_hierarchy[task_id].get('children', [])
+        for child_id in children:
+            result.update(self._get_all_tasks_in_hierarchy(child_id))
+
+        return result
+
+    def _get_task_tree_string(
```

Pylint (3.3.7)

[refactor] Too many local variables (27/15)

(R0914)